lib/elixir/test/elixir/gen_server_test.exs

```diff
@@ -50,8 +55,17 @@ defmodule GenServerTest do
     assert GenServer.cast(:foo, {:push, :world}) == :ok
   end
 
-  test "call/3 exit messages" do
+  test "call/3 exit messages when calling self" do
     assert catch_exit(GenServer.call(self, :pop, 5000)) == {:calling_self, {GenServer, :call, [self, :pop, 5000]}}
+
```

Comment:
The indirection is a bit confusing here, can we just copy the format of the above test? We don't really gain anything from making the call inside an actual GenServer.